src/ralph/back_office/models.py

```diff
@@ -41,7 +47,92 @@ class Meta:
 
     @property
     def country_code(self):
-        return 'PL'
+        if self.owner:
+            iso2 = Country.name_from_id(int(self.owner.country)).upper()
+            return iso2_to_iso3(iso2)
+        return 'POL'
```

Comment:
i know it's already merged but:
* why country code couldn't be null/None? 
* what if owner doesn't have country specified? you'll probably get some ValueError here (IMO we should return None then)